pkg/apis/operator/v1alpha1/tektonhub_types.go

```diff
@@ -120,6 +120,10 @@ type TektonHubStatus struct {
 	// The current installer set name
 	// +optional
 	HubInstallerSet map[string]string `json:"hubInstallerSets,omitempty"`
+
+	// The Base64 Encode URI of the Custom Logo
+	// +optional
+	Custom_Logo_Base64_URI string `json:"customLogoUri,omitempty"`
```

Comment:
we are not adding any default for this, so if nothing is provided, what logo will be shown, as config map is using the value of this field

Reply:
in case if nothing is provided then default Hub logo will be shown